### PR TITLE
webdb-ii-challenge

### PR DESCRIPTION
**Pull request by vijaydas**

### CODE REVIEW

**Comment by vijaydas** [comment]
Add Bears stretch

**Comment by vijaydas** [comment]
add separate knexfile

File: api/server.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+
+const port = 3300;
+server.listen(port, function() {
+  console.log(`\n=== Web API Listening on http://localhost:${port} ===\n`);
+});
+
+
+module.exports = server;
```
Comment (angeltorgon) [inline api/server.js:45]: ✔️

File: bears/bearModel.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    .update(changes)
+}
+
+function remove(id) {
+    return db('bears')
+    .where({id})
+    .del()
+}
+
+
```
Comment (angeltorgon) [inline bears/bearModel.js:43]: ✔️

File: bears/bearRouter.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  } 
+  catch(err) {
+    res.status(500).json({message: "fail"})
+  }
+ 
+}
+
+
+
+
```
Comment (angeltorgon) [inline bears/bearRouter.js:77]: ✔️

File: zoo/zooRouter.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  catch(err) {
+    res.status(500).json({message: "fail"})
+  }
+ 
+}
+
+
+
+
+module.exports = router;
```
Comment (angeltorgon) [inline zoo/zooRouter.js:102]: ✔️

**Review by angeltorgon — COMMENTED** [review]
## Minimum Viable Product
* [x] Configure 'knex' to connect to the '/data/lambda.db3'
* [x] Api endpoints

## STRETCH PROBLEM
* [x] Bears table and endpoints 

# Project Manager Feedback
## Great
* You completed all MVP objectives! :heavy_check_mark:
* You completed stretch objectives! :heavy_check_mark:
Great job Vijay!
## Requested Improvements
* Try adding seeds and migrations to your database!
## Questions
* How comfortable to you feel configuring knex?
## Rating {3}
:star: :star: :star:

**Pull request merged**